integreat_cms/release_notes/current/unreleased/3144.yml

```
@@ -0,0 +1,2 @@
+en: Fix the bug that pages cannot be deleted even with no child page
+de: Behebe den Fehler, dass Seiten nicht gelöscht werden können, auch wenn keine untergeordnete Seite vorhanden ist
```

```suggestion
de: Behebe den Fehler, dass Seiten nicht gelöscht werden können, auch wenn keine untergeordnete Seite mehr vorhanden ist
```
This is a super nit: I was just thinking that the current release note sounds like deleting is never possible, but for pages that never had a child page deleting works (at least for me). That's why I suggest to use a wording that makes it clearer that deleting didn't work if at any point there was a child page. But as mentioned in the intro: It's not highly relevant :)